### DISCUSSION TITLE
How to access the database/sql of Directus for Docker type and transfer to non Docker db?

### DISCUSSION THREAD

**Discussion by suhaimim** [directus/directus#9970] [Archive]
Hi,

May I know how to access the database from docker type of directus?
I couldn't find the database setting on the Directus setting interface.
![image](https://user-images.githubusercontent.com/35832684/142822214-fdb2c701-d7af-494d-a18c-c3bb527d42b0.png)

**Comment by rijkvanzanten** [comment]
If you haven't configured any database explicitly through the DB_* environment variables, the Docker image will default to SQLite. The database shown in your screenshot is the SQLite database used in the image 👍🏻

**Reply by suhaimim** [reply]
Yes, understand. The default docker image pulled already set the db and I don't know how to change/modify it.

**Reply by rijkvanzanten** [reply]
That file you showed in the screenshot is the whole database. You can modify that using any SQLite compatible tool 👍🏻 

For ease of development, I'd recommend copying the file from the container to the host so you don't accidentally lose any changes if/when the container is recreated.

**Reply by suhaimim** [reply]
Yes, I am planning to copy/move the db to local host instead of running inside the container, but I am still not able to find the way yet.
Appreciate if you could guide me. Thank you.

**Reply by azrikahar** [reply]
The sample [docker-compose.yml](https://docs.directus.io/getting-started/installation/docker/#docker-compose) in the docs does mention about this:

>     volumes:
>         ...
>         # Make sure to also mount the volume when using SQLite
>         # - ./database:/directus/database

Note: also recommend to check the other volumes mentioned such as `uploads` and `extensions` whether you need to persist them or not in your case.

You can also refer to the Docker official docs to understand more as this is pertains more to Docker/Docker Compose: https://docs.docker.com/compose/compose-file/compose-file-v3/#volumes, particular the Short Syntax section which is used in the sample compose file.
